apps/crypto_test/src/main.c

```diff
@@ -286,7 +288,10 @@ run_benchmark(char *name, block_encrypt_func_t encfn, void *data, uint8_t iter)
             }
         }
     }
-    printf("done in %lu ticks\n", os_time_get() - t);
+    e = os_time_get() - t;
+    ret = os_time_ticks_to_ms(e, &ms);
+    assert(ret == 0);
+    printf("done in %lu ticks / %lu ms\n", e, ms);
```

Comment:
When **RISCV** (unlike ARM) gcc compiler is used, **uint32_t** is not `unsigned long int` but just `unsigned int`.
Usage of `"%lu"` will lead to warnings there.
I would suggest to use **PRIu32** or cast to avoid future warning fixes.

Reply:
done.